### DISCUSSION TITLE
Creating a local network with multiple miner nodes

### DISCUSSION THREAD

**Discussion by mrihn** [filecoin-project/lotus#8247] [General Q&A] [answered]
This is not a bug or feature request. Just a general question:
How does one add a second lotus-miner node to an existing local network?

I followed the instructions for setting up a local network here:
https://lotus.filecoin.io/docs/developers/local-network/

I am able to start up the lotus daemon and lotus-miner node without any errors, but it is not clear to me how to add a new miner node once the network is up (or how to start up a network with multiple miner nodes)

I tried to add a second miner node by creating a new pre-seal-t01001.json file and adding it to the localnet.json file with this command:
`./lotus-seed genesis add-miner localnet.json ~/.genesis-sectors/pre-seal-t01001.json`

but when I try to initialize it with the `./lotus-miner init` command, I get this error:
"ERROR: repo at '/root/.lotus-miner-local-net' is already initialized"

lotus version 1.15.1-dev+2k+git.6291c0c58.dirty

**Comment by LexLuthr** [comment, marked as answer]
@mrihn You need to use a separate directory for the new local miner and also make sure that any output generated during the install and configure phase are separate from the first miner you created.
1. Create new miner path i.e. export LOTUS_MINER_PATH=~/.lotus-miner-local-net2
2. Specify a new miner address as default would already be in use.
3. Seal new genesis sectors at a different location.
4. Make sure to use a new localnet.json as well. 

If anything is shared between the 2 miner except the lotus node, you will end up with an error.
